### ISSUE TITLE
Can not jailbreak

### ISSUE THREAD

**Issue by Far88han** [checkra1n/BugTracker#2254]
**Tell us about your setup:**
1. What iDevice are you using?
2. On what version of iOS is it?
3. What version of checkra1n are you using?
4. What is your host system (OS version? Hackintosh? VM? etc.)?
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?

**What are the steps to reproduce the issue?**
1. Iphone8
2. 14.5
3.0.124
4.window
5.usb type lighting
...

**What do you expect, and what is happening instead?**
-i can not jailbreak..after successfully booting,home scren out the apple icon and the my device going to restart

please help me,what i have to do

**Comment by Far88han** [comment]
> **Tell us about your setup:**
> 
> 1. What iDevice are you using?
> 2. On what version of iOS is it?
> 3. What version of checkra1n are you using?
> 4. What is your host system (OS version? Hackintosh? VM? etc.)?
> 5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
> 
> **What are the steps to reproduce the issue?**
> 
> 1. Iphone8
> 2. 14.5
>    3.0.124
>    4.window
>    5.usb type lighting
>    ...
> 
> **What do you expect, and what is happening instead?** -i can not jailbreak..after successfully booting,home scren out the apple icon and the my device going to restart
> 
> please help me,what i have to do

**Comment by adhip94** [comment]
Yes, even i am having this issue, I just upgraded to iOS 14.8 and I am not able to jailbreak, I read many issues saying that usb-c/lightning doesnt work and to use USB-A instead.

**Comment by theg1l** [comment]
latest checkra1n only support the ios ver till 14.5 i think. cmiiw. so you have to wait for another updates ver of checkra1n (like me)
